### DISCUSSION TITLE
How to inline the CSS content with HTML

### DISCUSSION THREAD

**Discussion by princescar** [vikejs/vike#2006] [Discussion]
It seems the max inline size option of vite is not applied to SSR. Is it possible to inline the content of generated CSS file into the server rendered HTML, to reduce the additional network requests for initial rendering?

**Comment by brillout** [comment]
In principle, it should work just fine if Vite returns a base64 data url instead of the the asset path upon `import someImage from './some-image.svg'`.

Contribution welcome to dig whether it's something that can be improved on Vite's side or on Vike's side.
